Simplified/NYPLMyBooksViewController.h

```diff
@@ -5,6 +5,8 @@
 - (id)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil NS_UNAVAILABLE;
 
+- (void)didSelectSync;
```

Comment:
Actually I take it back. If you are switching libraries while in My Books, then there's no reason to sync the registry. That behavior can stay as is + the end of account auth. So I'm undoing my change here.